pkg/cmd/cli/cmd/deploy.go

```diff
@@ -221,10 +221,10 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 		}
 	}
 
-	config.LatestVersion++
+	config.Status.LatestVersion++
```

Comment:
To be clear, I'm not asking for a change in this pull.  I see it as a follow-up issue.